src/test/java/uk/gov/hmcts/reform/notificationservice/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.notificationservice.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.notificationservice.data.NotificationRepository;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class NotificationServiceTest {
+
+    @Mock
+    private NotificationRepository notificationRepository;
+
+    @Test
+    void should_call_repository_for_pending_notifications_to_begin_the_process() {
+        getNotificationService().processPendingNotifications();
+
+        verify(notificationRepository, times(1)).findPending();
+    }
+
+    private NotificationService getNotificationService() {
+        return new NotificationService(notificationRepository);
+    }
```

Comment:
I assume this is in preparation for future tests. But can we just initialize it in test setup method?

Reply:
from last PRs i saw a cheerful reactions of key components not being in setup method but just as separate one. if not then i'm mixing with smth else

Reply:
A separate method is really great when there are cases in which tested class can have different dependencies/parameters, depending on the scenario.